elide-async/src/test/java/com/yahoo/elide/async/hooks/ExecuteQueryHookTest.java

```diff
@@ -66,22 +66,6 @@ public void testWithResultFormatTypeNull() {
         });
     }
 
-    @Test
```

Comment:
Do we already have a test to assert the default behavior?

Reply:
Added one now.